email_processing.py

```diff
@@ -0,0 +1,197 @@
+import requests
+import datetime
+import time
+import os
+import smtplib
+from email.message import EmailMessage
+from requests.exceptions import ConnectionError
+
+# create a class to communicate with RossumAPI
+class RossumAPI(object):
+    ''''
+    Script to check Rossum processing of e-mail attachments sent to an organization inbox.
+        1. script performs authentication at Rossum API using get_access_token().
+        2. sends an email with a file attachment (FILE_NAME) to an email address of a trial organization (EMAIL)
+        3. waits for the document to be processed and checks that processing finished OK (annotation status is “to_review”).
+
+        A Python script uses environment variables:
+
+        • FILE_NAME -- file to send as an attachment
+        • EMAIL -- Rossum inbox address
+        • API_URL -- e.g. api.elis.rossum.ai
+        • ROSSUM_USER -- username of a user in trial organization
+        • ROSSUM_PASSWORD -- password of a user in trial organization
+
+        If email was processed correctly, file was loaded and shown in API -> return 0
+        If email was not processed correctly or file was not found in API or server cannot be reached -> return 1
+    '''
+    access_token = None
+    all_files = []
+    # ------------- START config
+#   Created environment variables according to task description:
+    ROSSUM_USER = os.environ.get('ROSSUM_USER')
+    ROSSUM_PASSWORD = os.environ.get('ROSSUM_PASSWORD')
+    FILE_NAME = os.environ.get('FILE_NAME')
+    API_URL = os.environ.get('API_URL')
+    EMAIL = os.environ.get('EMAIL')
+
+    # My own variables used for authentication at smtp.gmail.com to send an email with attachment
+    # Needs to have configured gmail account (gmail smtp is used in import_by_email function) to send email to EMAIL address of trial organization
+    SENDER_EMAIL_ADDRESS = os.environ.get('SENDER_EMAIL_ADDRESS')
+    SENDER_EMAIL_PASSWORD = os.environ.get('SENDER_EMAIL_PASSWORD')
+
+    # -------------- END config
+
+    def get_login_data(self):
+        return {
+            'username': self.ROSSUM_USER,
+            'password': self.ROSSUM_PASSWORD,
+        }
+
+    def get_access_token(self):
+        user = self.ROSSUM_USER
+
+        login_api_url = f'https://{self.API_URL}/v1/auth/login'
+        login_data = self.get_login_data()
+
+        print('Performing authentication at RossumAPI ... ')
+        try:
+            r = requests.post(login_api_url, data=login_data)
+            # Get HTTP status code of a domain + path + data
+            if r.status_code not in range(200, 299):
+                raise Exception("Could not authenticate client.")
```

Comment:
I'd just let the `requests` raise an error
```suggestion
            r = requests.post(login_api_url, data=login_data)
            r.raise_for_status()
```